core/unit_test/view/TestMDSpanConversion.hpp

```diff
@@ -0,0 +1,393 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <gtest/gtest.h>
+#include <type_traits>
+
+#include <Kokkos_Core.hpp>
+
+#ifdef KOKKOS_ENABLE_IMPL_MDSPAN
+
+template <class T, class ExecutionSpace>
+struct TestViewMDSpanConversion {
+  using value_type = T;
+
+  template <std::size_t Padding>
+  using layout_left_padded = Kokkos::Experimental::layout_left_padded<Padding>;
+
+  template <std::size_t Padding>
+  using layout_right_padded =
+      Kokkos::Experimental::layout_right_padded<Padding>;
+
+  struct test_accessor {
+    using offset_policy    = test_accessor;
+    using element_type     = value_type;
+    using reference        = element_type &;
+    using data_handle_type = element_type *;
+
+    constexpr test_accessor() noexcept = default;
```

Comment:
Any good reason why you added the noexcept specific to the constructor but omitted it for the member functions below?
If not please add it below for consistence with `std::default_accessor`.

Reply:
No, was an oversight